### PR TITLE
upgrade storybook to 5.0.7

### PR DESCRIPTION
**Pull request by cellog**
# Description

This PR *finally* upgrades us to storybook version 5. It fixes the issue by duplicating the .babelrc from the root directory into `storybook/.babelrc`. This is clearly a bug in storybook, which is reported over [here](https://github.com/storybooks/storybook/issues/5882).

Additional changes needed to make storybook work in `unlock-app` and `unlock-protocol` were updating the syntax of the `webpack.config.js` to match the new format.

Screen Shot 2019-04-14 at 9 37 31 PM
Screen Shot 2019-04-14 at 9 37 28 PM
Screen Shot 2019-04-14 at 9 37 25 PM

# Issues

Fixes #2635 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by cellog** [comment]
ah! I forgot tickets, adding that in a second

**Comment by cellog** [comment]
Screen Shot 2019-04-14 at 9 56 39 PM

**Comment by julien51** [comment]
Yeah the tickets footer is not great :/ I need to fix it!

File: paywall/next.config.js [hunk 1]
```diff
@@ -32,6 +32,23 @@ Object.keys(requiredConfigVariables).forEach(configVariableName => {
   }
 })
 
+// TODO: remove this when next gets their act together
+// see https://github.com/zeit/next-plugins/issues/392#issuecomment-475845330
+function HACK_removeMinimizeOptionFromCssLoaders(config) {
```
Comment (julien51) [inline paywall/next.config.js]: 😭

**Pull request merged**